_data/project.yml

```diff
@@ -87,3 +87,21 @@
   badge3: Python
   description: Displaying statistics on NHS and NHSX-affiliated publications on Pubmed.
   author: Martina Fonseca
+
+- title: SynthVAE
+  link: https://nhsx.github.io/nhsx-publications/
```

Comment:
Link needs updating to `https://github.com/nhsx/SynthVAE`

Reply:
Good spot.  Was sure I had done that. Commit made.